include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1455,6 +1455,18 @@ class MatrixFree : public Subscriptor
                     const unsigned int vector_number,
                     const unsigned int dof_handler_index = 0) const;
 
+  /**
+   * This returns the pair of the cell level and cell index, that is
+   * <tt>cell->level()</tt> and <tt>cell->index()</tt>, given a
+   * TriaIterator <tt>cell</tt>, to a given cell in the renumbering of
+   * this structure.
+   *
+   * See get_cell_iterator() for more information.
```

Comment:
That sounds pretty convoluted. What about referring to `get_cell_iterator()` similar to
"This returns level and index for the cell that would be returned by get_cell_iterator() when called with the same arguments."?

Reply:
Thanks. Fixed it.